src/ProgramBlockEditor.js

```diff
@@ -343,11 +427,12 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
                     expandedMode={this.props.addNodeExpandedMode}
                     isDraggingCommand={this.props.isDraggingCommand}
                     programStepNumber={programStepNumber}
+                    closestAddNodeIndex={this.state.closestAddNodeIndex}
                     disabled={
                         this.props.editingDisabled ||
                         (!this.commandIsSelected() && !this.props.isDraggingCommand)}
                     onClick={this.handleClickAddNode}
-                    onDrop={this.handleDropCommand}
+                    onDrop={this.handleDropCommandOnAddNode}
```

Comment:
The `onDrop` callback is no longer needed.

Reply:
Done.